test/e2e/dra/dra.go

```diff
@@ -775,6 +777,70 @@ var _ = ginkgo.Describe("[sig-node] DRA [Feature:DynamicResourceAllocation]", fu
 	})
 })
 
+// This test covers aspects of non graceful node shutdown by DRA controller
+// More details about this can be found in the KEP:
+// https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2268-non-graceful-shutdown
+var _ = ginkgo.Describe("[sig-node] [Serial] [Disruptive] [Slow] [Feature:DynamicResourceAllocation] DRA: handling non graceful node shutdown", func() {
```

Comment:
Does this have to be a new `Describe`? Can't it be a new `It` under some existing context?

Reply:
No, it doesn't. I separated it because I wanted to show that's this is a special test case. It's disruptive, it shuts down test node, it works only with kind. I think it deserves to be visually separated in the test code.

Reply:
Not sure whether I agree with this rationale. It has a few downsides (some duplicated setup source code lines, full test name.

What is the the full test name?

Reply:
Let me be more explicit "DRA: handling non graceful node shutdown cluster must deallocate on non graceful node shutdown" just doesn't look right.